### ISSUE TITLE
[Bug] ginkgo65 keymap incorrect

### ISSUE THREAD

**Issue by nicedream06** [qmk/qmk_firmware#19584]
### Describe the Bug

The bug in issue 17307 that affected the gingko65hot keymap also seems to affect the gingko65 (solderable) keymap.

After flashing, most keys are inoperable.
The few that do work register the wrong keys.

Examples:
- Esc = Left Ctrl
- 1 = Left Win
- Down Arrow = Backspace
- Left Arrow = +/=

Link to the similar bug on gingko65hot
https://github.com/qmk/qmk_firmware/issues/17307

### Keyboard Used

gingko65

### Is AutoHotKey / Karabiner installed

- [ ] AutoHotKey (Windows)
- [ ] Karabiner (macOS)

**Comment by nicedream06** [comment]
Did some testing.  It seems that the following lines in config.h will fix the issue:

`#define MATRIX_ROW_PINS { B0, B1, B2, B3, F7 }`
`#define MATRIX_COL_PINS { C7, F6, F5, F4, F1, E6, D0, D1, D2, D3, D5, D4, D6, D7, B4, B5 }`

**Comment by waffle87** [comment]
Yeah looks like the consequences of copy/paste again. Feel free to open a PR or say otherwise if you'd like someone else to.

**Comment by nicedream06** [comment]
I'll let someone else do the pull request. I don't really mess around in github that much, so I'm kind of inexperienced. I just wanted to report the bug I found, and then the fix that worked for me.
